dndwiki/classes/class-gunslinger.xml

```diff
@@ -0,0 +1,874 @@
+      <p><strong>Spell save DC</strong> = 8 + your proficiency bonus + your Charisma modifier</p>
+      <p><strong>Spell attack modifier</strong> = your proficiency bonus + your Charisma modifier</p>
+    </description>
+    <sheet>
+      <description>You can use a holy symbol as a spellcasting focus for your paladin spells.</description>
+    </sheet>
+    <spellcasting name="Preacher" ability="Charisma" prepare="false" allowReplace="true">
+      <list>Cleric, Paladin</list>
+    </spellcasting>
+    <rules>
+      <grant type="Grants" id="ID_INTERNAL_GRANT_MULTICLASS_SPELLCASTING_SLOTS_THIRD" requirements="ID_INTERNAL_GRANT_MULTICLASS"/>
+
+      <stat name="Preacher:spellcasting:slots:1" value="2" level="3" />
+      <stat name="Preacher:spellcasting:slots:1" value="1" level="4" />
+      <stat name="Preacher:spellcasting:slots:1" value="1" level="7" />
+      <stat name="Preacher:spellcasting:slots:2" value="2" level="7" />
+      <stat name="Preacher:spellcasting:slots:2" value="1" level="10" />
+      <stat name="Preacher:spellcasting:slots:3" value="2" level="13" />
+      <stat name="Preacher:spellcasting:slots:3" value="1" level="16" />
+      <stat name="Preacher:spellcasting:slots:4" value="1" level="19" />
+
+      <select type="Spell" name="Cantrip (Preacher)" supports="Cleric, 0" level="3" number="2" spellcasting="Preacher" />
+      <select type="Spell" name="Cantrip (Preacher)" supports="Cleric, 0" level="10" number="1" spellcasting="Preacher" />
+
+      <select type="Spell" name="Spellcasting (Preacher)" supports="Paladin, 1" level="3" number="3" spellcasting="Preacher" />
+      <select type="Spell" name="Spellcasting (Preacher)" supports="Paladin, 1" level="4" spellcasting="Preacher"/>
+      <select type="Spell" name="Spellcasting (Preacher)" supports="Paladin, (1 || 2)" level="7" spellcasting="Preacher"/>
+      <select type="Spell" name="Spellcasting (Preacher)" supports="(Paladin || Cleric), (1 || 2)" level="8" spellcasting="Preacher"/>
+      <select type="Spell" name="Spellcasting (Preacher)" supports="Paladin, (1 || 2)" level="10" spellcasting="Preacher"/>
+      <select type="Spell" name="Spellcasting (Preacher)" supports="Paladin, (1 || 2)" level="11" spellcasting="Preacher"/>
+      <select type="Spell" name="Spellcasting (Preacher)" supports="Paladin, (1 || 2 || 3)" level="13" spellcasting="Preacher"/>
+      <select type="Spell" name="Spellcasting (Preacher)" supports="(Paladin || Cleric), (1 || 2 || 3)" level="14" spellcasting="Preacher"/>
+      <select type="Spell" name="Spellcasting (Preacher)" supports="Paladin, (1 || 2 || 3)" level="16" spellcasting="Preacher"/>
+      <select type="Spell" name="Spellcasting (Preacher)" supports="Paladin, (1 || 2 || 3 || 4)" level="19" spellcasting="Preacher"/>
+      <select type="Spell" name="Spellcasting (Preacher)" supports="(Paladin || Cleric), (1 || 2 || 3 || 4)" level="20" spellcasting="Preacher"/>
```

Comment:
Cantrips are correct, but Spells aren't. First of all, `supports="1"` will restrict this select to that spell level, which means when you try to replace that spell you will be able to select only from level 1 spells, which is not how replacement works, so they all should be replaced with `$(spellcasting:slots)`.
You probably should also replace all `Paladin` supports with `$(spellcasting:list)` (you can see the same thing in Eldritch Knight spellcasting)